site/src/pages/ManagementSettingsPage/OrganizationSettingsPage.test.tsx

```diff
@@ -122,4 +122,24 @@ describe("OrganizationSettingsPage", () => {
     await renderPage("the-endless-void");
     await screen.findByText("Organization not found");
   });
+
+  it("cannot edit organization", async () => {
+    server.use(
+      http.get("/api/v2/organizations", () => {
+        return HttpResponse.json([MockDefaultOrganization]);
+      }),
+      http.post("/api/v2/authcheck", async () => {
+        return HttpResponse.json({
+          viewDeploymentValues: true,
+        });
+      }),
+    );
+    // No form since they cannot edit, instead sees the summary view.
+    await renderPage(MockDefaultOrganization.name);
+    expect(screen.queryByTestId("org-settings-form")).not.toBeInTheDocument();
+    await screen.findByRole("heading", {
+      level: 1,
+      name: MockDefaultOrganization.display_name,
+    });
+  });
```

Comment:
this is the sort of thing that would be much better off as a storybook test. `[data-testid="org-settings-form"]` could not be in the document for all sorts of reasons. someone refactored and this attribute got lost in the shuffle, someone changed the attribute value but forgot to do a find-and-replace so the test is now out of date, etc. I'd just make a story and let chromatic snapshot it.

Reply:
I agree, but since this happens outside the view it seems tough to do, unless we have a way to mock API calls in Storybook (honestly no idea).

I could move the code into the settings view, but that felt wrong too.  :thinking: 

Alternatively, maybe a new super-view like `OrganizationSummaryOrSettingsPageView`?

Reply:
Or maybe I am overthinking it, and doing it in `OrganizationSettingsPageView` is not that bad

Reply:
you can mock queries: https://github.com/coder/coder/blob/21942afef362b0e8f2d7ccfeaff10293d5cd25d5/site/src/pages/WorkspacePage/WorkspaceTopbar.stories.tsx#L266-L276

Reply:
Ahhh thank you this will teach me to make assumptions  :stuck_out_tongue:

Reply:
Oof and frontend.md describes everything too, definitely a case of rtfm :facepalm:

Reply:
it happens lol